tests/integration/callbacks/test_long_callback.py

```diff
@@ -0,0 +1,311 @@
+from multiprocessing import Lock, Value
+import time
+
+import dash_core_components as dcc
+import dash_html_components as html
+import dash
+from dash.dependencies import Input, Output, State
+
+
+def test_lcb001_fast_input(dash_duo, diskcache_manager):
```

Comment:
not a big deal, but the pattern we typically use is 4 characters, 2 from the folder and 2 from the file, so this would be `cblc001`